assets/src/components/detourListPage.tsx

```diff
@@ -34,7 +34,7 @@ export const DetourListPage = () => {
       if (isErr(snapshot)) {
         return undefined
       }
-      return snapshot.ok
+      return detourResponse
```

Comment:
```suggestion
      return detourResponse.ok
```

Then on lines +92 to +98
```tsx
          {...(detour
            ? {
                snapshot: detour.state,
                author: detour.author,
                updatedAt: detour.updatedAt,
              }
            : {})}
```

Reply:
Ha! Totally, thanks ✅